lib/miq_automation_engine/service_models/miq_ae_service_container.rb

```diff
@@ -0,0 +1,4 @@
+module MiqAeMethodService
+  class MiqAeServiceContainer < MiqAeServiceModelBase
```

Comment:
@mkanoor I want to expose some of the relationships.  Thank you for the tip.  I'll update the pull request in the next day or two.  Or would you rather me close this request and open a new one?